package.json

```diff
@@ -48,17 +48,18 @@
   },
   "homepage": "https://github.com/ng-bootstrap/ng-bootstrap#readme",
   "dependencies": {
-    "@angular/core": "4.0.3",
-    "@angular/common": "4.0.3",
-    "@angular/forms": "4.0.3"
+    "@angular/animations": "^4.3.5",
```

Comment:
I noted the breaking change in a separate commit: https://github.com/ng-bootstrap/ng-bootstrap/pull/1765/commits/61109dd45a0da30eba171d44333afd25d5484ca6 (I kept it this way so it could be cherry-picked to another branch and used to add additional animations in a further PR).

Looking at the change log, a minimum of 4.2.0 is required, however there have been a lot of bug fixes in future versions so maybe that's higher in reality. I will try it out with 4.2.0 and see if it works ok.